javaslang/src/main/java/javaslang/collection/Multimap.java

```diff
@@ -551,8 +550,7 @@ default int length() {
     @Override
     default <U> Seq<U> flatMap(Function<? super Tuple2<K, V>, ? extends Iterable<? extends U>> mapper) {
         Objects.requireNonNull(mapper, "mapper is null");
-        // don't remove cast, doesn't compile in Eclipse without it
-        return (Seq<U>) iterator().flatMap(mapper).toStream();
+        return Stream.ofAll(iterator()).flatMap(mapper);
```

Comment:
Also here: `return (Seq<U>) iterator().flatMap(mapper).toStream();` is already the best we can do!